backend/src/common/types/storage/storage-upload-file-request-dto.type.ts

```diff
@@ -0,0 +1,7 @@
+type StorageUploadFileRequestDto = {
+  space: string;
```

Comment:
```suggestion
  string: string;
```

Reply:
@what1s1ove I'm a bit confused, I think you meant to name a property in a different way, because `string` doesn't tell anything about the property

Reply:
oh sorry. should be a `bucket`